pkg/runtime/runtime.go

```diff
@@ -2951,6 +2958,64 @@ func (a *DaprRuntime) createAppChannel() (err error) {
 	return nil
 }
 
+// Returns the HTTP endpoint for the app.
+func (a *DaprRuntime) getAppHTTPEndpoint() string {
+	port := strconv.Itoa(a.runtimeConfig.ApplicationPort)
+	if a.runtimeConfig.AppSSL {
+		return "https://" + channel.DefaultChannelAddress + ":" + port
+	} else {
+		return "http://" + channel.DefaultChannelAddress + ":" + port
+	}
+}
+
+// Initializes the appHTTPClient property.
+func (a *DaprRuntime) initAppHTTPClient() {
+	var tlsConfig *tls.Config
+	if a.runtimeConfig.AppSSL {
+		tlsConfig = &tls.Config{
+			//nolint:gosec
+			InsecureSkipVerify: true,
+			// For 1.11
+			// MinVersion:         channel.AppChannelMinTLSVersion,
+		}
+		// For 1.11
```

Comment:
Reviewers: when I rebased from master to 1.10, there was a merge conflict. Want to make sure this stays here for when we merge the release branch back into master, so we don't miss this.